src/main/java/com/cincialert/controllers/CinciAlertController.java

```diff
@@ -0,0 +1,24 @@
+package com.cincialert.controllers;
```

Comment:
Its fine if you want to have custom controllers for each entity, but you should still have routing for the project itself. I added this controller to manage startup and routing between non-model processes